src/imb_collective.jl

```diff
@@ -26,7 +26,7 @@ function run_imb_collective(benchmark::MPIBenchmark, func::Function, conf::Confi
 
     for s in conf.lengths
         size = 1 << s
-        iters = conf.iters(conf.T, s)
+        isa(conf.iters, Function) == true ?  iters = conf.iters(conf.T, s) : iters = conf.iters
```

Comment:
Can assign `iters` to the output of the ternary operator instead of doing assignments inside the ternary operator.  Also, no need to check that a boolean is `== true`, you can directly use its value 🙂 
```suggestion
        iters = conf.iters isa Function ?  conf.iters(conf.T, s) : conf.iters
```

Same below.

Reply:
Thank you.